src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/task/BehandlingsstatistikkTask.kt

```diff
@@ -142,13 +143,10 @@ class BehandlingsstatistikkTask(private val iverksettClient: IverksettClient,
                             hendelseTidspunkt = LocalDateTime.now(),
                             gjeldendeSaksbehandler = SikkerhetContext.hentSaksbehandler(true))
 
-        fun opprettVedtattTask(behandlingId: UUID,
-                               hendelseTidspunkt: LocalDateTime,
-                               oppgaveId: Long?): Task =
+        fun opprettVedtattTask(behandlingId: UUID): Task =
                 opprettTask(behandlingId = behandlingId,
                             hendelse = Hendelse.VEDTATT,
-                            hendelseTidspunkt = hendelseTidspunkt,
-                            oppgaveId = oppgaveId)
+                            hendelseTidspunkt = LocalDateTime.now())
```

Comment:
Tolker jeg det riktig att denne vedtattTask kommer peke til oppgaveId til oppgaven som blir opprettet i `opprettGodkjennVedtakOppgave` i stedet for den oppgaven som var for selve behandlingen? 
Er det riktig? Og er det ønskelig?

Reply:
Godt spørsmål - kan diskutere det i morgen. Er jo en fare for at dét skjer, men poenget med den oppgaveId-en er å fange opp riktig enhet, tror jeg? 🤔